File: Codeforces/Round #501/A.cpp
#include <iostream>
#include <cstdio>
#include <cstring>
using namespace std;
int n, m;
bool vis[105];
void input() {
    cin >> n >> m;
    while (n--) {
        int l, r;
        cin >> l >> r;
        for (int i = l; i <= r; ++i)    vis[i] = 1;
    }
}
void output() {
    int ans = 0;
    for (int i = 1; i <= m; ++i) if (vis[i] == 0)   ++ans;
    cout << ans << endl;
    for (int i = 1; i <= m; ++i) if (vis[i] == 0)   cout << i << ' ';
}
int main() {
    ios::sync_with_stdio(false);
    cin.tie(0);
    input();
    output();
}

File: Codeforces/Round #501/A.py
n, m = map(int, input().split())
ans = list(range(1 + m))
for i in range(n):
    l, r = map(int, input().split())
    for j in range(l, r + 1):
        ans[j] = 0
b = []
for i in range(1 + m):
    if ans[i] != 0:
        b.append(i)

print(len(b))
print(*b)